### DISCUSSION TITLE
Creating small tasks (todo lists) inside the PRs

### DISCUSSION THREAD

**Discussion by tawmoto** [community/community#55041] [Pull Requests]
Hi,

In ButBucket there is very nice option regarding the PRs. When I open a PR, a reviewer can comment something, I click on that comment and "Convert (comment to) task". After doing this with many comments, I have a visible personal tasklist which I have to do before I merge. So I don't longer have to search through all of the comments on a pull request to find follow-up items.

I don't see anything like this in GitHub, the only similar option seems that the reviewer (not me) can "request change", which is kinda strange, I want me to create that task list, not the reviewer.

How can I do smth like this in GitHub?
thanks

![image](https://github.com/community/community/assets/35702340/dfbdb941-64dd-47e9-a366-fc74d56771ba)

![image](https://github.com/community/community/assets/35702340/24cc87a7-50d5-42e5-aeb4-d784e4ff6476)

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @tawmoto ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by tawmoto** [reply]
> Thanks for posting in the GitHub Community, @tawmoto !
> 
> We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

Thank you but i did not find any category for this issue. Can you move it pls?
Thank you

**Reply by queenofcorgis** [reply]
@tawmoto I moved it to the applicable category Pull Requests. If there isn't a category for a post, please use General 😄 :octocat:
